models/articles-model.js

```diff
@@ -0,0 +1,12 @@
+const db = require("../db/connection.js");
+
+exports.selectArticle = (article_id) => {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
+      .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "article does not exist" });
+      }
+      return rows;
```

Comment:
Remember that the result is an array so we need to specifically grab index 0 for the single article